microsoft-edge/devtools-guide-chromium/css/grid.md

```diff
@@ -126,46 +139,74 @@ The row line labels display only computed sizes, since there are no row sizes de
 | **80px** | &nbsp;| 80px |
 | **80px** | &nbsp;| 80px |
```

Comment:
80px is wrong/outdated.
live article section:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid#show-track-sizes

PR article section: 
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid?branch=pr-en-us-3248#show-track-sizes

result:
```
The line labels on the rows display only computed sizes, since there are no row sizes defined in the stylesheet:

| Track size | Authored size | Computed size |
|:--- |:--- |:--- |
| **40px** | &nbsp;| 40px |
| **40px** | &nbsp;| 40px |
```

Reply:
i made that change but entered global comment to check all text/values to catch up w/ latest pngs